test-network-function/identifiers/identifiers.go

```diff
@@ -572,16 +577,26 @@ the changes for you.`,
 		Remediation:           `You should recreate the node or change the sysctls, recreating is recommended because there might be other unknown changes`,
 		BestPracticeReference: bestPracticeDocV1dot2URL + " Section 6.2",
 	},
-	TestScalingIdentifier: {
-		Identifier: TestScalingIdentifier,
+	TestDeploymentScalingIdentifier: {
+		Identifier: TestDeploymentScalingIdentifier,
 		Type:       normativeResult,
-		Description: formDescription(TestScalingIdentifier,
+		Description: formDescription(TestDeploymentScalingIdentifier,
 			`tests that CNF deployments support scale in/out operations. 
 			First, The test starts getting the current replicaCount (N) of the deployment/s with the Pod Under Test. Then, it executes the 
 			scale-in oc command for (N-1) replicas. Lastly, it executes the scale-out oc command, restoring the original replicaCount of the deployment/s.`),
 		Remediation:           `Make sure CNF deployments/replica sets can scale in/out successfully.`,
 		BestPracticeReference: bestPracticeDocV1dot2URL + " Section 6.2",
 	},
+	TestStateFulSetScalingIdentifier: {
+		Identifier: TestStateFulSetScalingIdentifier,
+		Type:       normativeResult,
+		Description: formDescription(TestStateFulSetScalingIdentifier,
+			`tests that CNF deployments support scale in/out operations. 
+			First, The test starts getting the current replicaCount (N) of the statefulset/s with the Pod Under Test. Then, it executes the 
```

Comment:
@greyerof  is the `with the Pod Under Test` part out of date? is it there because we used to find deployments through the pod list as part of the test? also, any suggestion on how to update the steps to reflect the HPA case?

Reply:
No, I think it's still a valid statement. Both target deployments and statefulsets are autodiscovered using the same targetPodLabels as we use for discovering the PUTs. We just search those labels on the deployment/statefulset's .spec.template.metadata.labels.